services/catalog/tests/test_endpoints.py

```diff
@@ -148,6 +148,29 @@ def test_post_search_in_toto_s1_l1_collection(self, client):  # pylint: disable=
         content = json.loads(response.content)
         assert len(content["features"]) == 2
 
+    def test_search_in_unexisting_collection(self, client):
```

Comment:
One question here, you are searching in collection that doesn't exist using a owner_id that **doesn't exist** "tata".

Shouldn't you test with an owner_id that exists "toto" in a collection that doesn't exist?

Reply:
Or even have both tests.

Reply:
I will add these tests thanks :+1: